src/uu/sort/src/sort.rs

```diff
@@ -45,6 +45,7 @@ use std::ops::Range;
 use std::path::Path;
 use std::path::PathBuf;
 use std::str::Utf8Error;
+use sysinfo::{RefreshKind, System, SystemExt};
```

Comment:
Would it be possible to do the same with a crate that we already using?
AFAIK, we aren't using sysinfo currently